config.go

```diff
@@ -143,11 +144,19 @@ func getEffectiveConfig(configRoot, owner, repo string) (*effectiveConfig, error
 }
 
 func buildGlobalEffectiveConfig(global *config) effectiveConfig {
+	origServerID := firstNonEmptyStringPtr(global.GrimServerID, global.GrimQueueName, &defaultGrimQueueName)
+	truncatedServerID := origServerID
+
+	if len(truncatedServerID) > 15 {
+		truncatedServerID = fmt.Sprintf("%s", origServerID[:15])
+	}
+
 	return effectiveConfig{
 		grimQueueName:   firstNonEmptyStringPtr(global.GrimQueueName, &defaultGrimQueueName),
 		resultRoot:      firstNonEmptyStringPtr(global.ResultRoot, &defaultResultRoot),
 		workspaceRoot:   firstNonEmptyStringPtr(global.WorkspaceRoot, &defaultWorkspaceRoot),
-		grimServerID:    firstNonEmptyStringPtr(global.GrimServerID, global.GrimQueueName, &defaultGrimQueueName),
+		grimServerID:    truncatedServerID,
```

Comment:
My own personal preference is to leave the config as it was created by the user.  And to store the truncated in a new value.  That way there is no confusion about what the user entered.

That said it is a nitpick and you don't have to change it.